test/JDBC/expected/sys_sql_logins-vu-verify.out

```diff
@@ -1,31 +1,99 @@
-USE master
+-- tsql
+SELECT * FROM dbo.sys_sql_logins_vu_prepare_view
+GO
+~~START~~
+varchar#!#nvarchar
+jdbc_user#!#SQL_LOGIN
+sys_sql_logins_vu_login1#!#SQL_LOGIN
+sys_sql_logins_vu_login2#!#SQL_LOGIN
+sys_sql_logins_vu_login3#!#SQL_LOGIN
+~~END~~
+
+
+-- reset the login password
+ALTER LOGIN sys_sql_logins_vu_login1 WITH PASSWORD = '123';
+GO
+
+-- reset the login password
+ALTER LOGIN sys_sql_logins_vu_login2 WITH PASSWORD = '123';
 GO
 
-SELECT * FROM sys.sql_logins
+-- reset the login password
+ALTER LOGIN sys_sql_logins_vu_login3 WITH PASSWORD = '123';
+GO
+
+-- tsql user=sys_sql_logins_vu_login1 password=123
+SELECT * FROM dbo.sys_sql_logins_vu_prepare_view
 GO
 ~~START~~
-varchar#!#int#!#varbinary#!#char#!#nvarchar#!#int#!#datetime#!#datetime#!#varchar#!#varchar#!#int#!#int#!#bit#!#bit#!#bit#!#varbinary
+varchar#!#nvarchar
+jdbc_user#!#SQL_LOGIN
+sys_sql_logins_vu_login1#!#SQL_LOGIN
 ~~END~~
 
 
-SELECT * FROM sys_sql_logins_vu_prepare_view
+-- tsql user=sys_sql_logins_vu_login2 password=123
+SELECT * FROM dbo.sys_sql_logins_vu_prepare_view
 GO
 ~~START~~
-varchar#!#int#!#varbinary#!#char#!#nvarchar#!#int#!#datetime#!#datetime#!#varchar#!#varchar#!#int#!#int#!#bit#!#bit#!#bit#!#varbinary
+varchar#!#nvarchar
+jdbc_user#!#SQL_LOGIN
+sys_sql_logins_vu_login2#!#SQL_LOGIN
 ~~END~~
 
 
-EXEC sys_sql_logins_vu_prepare_proc
+-- tsql user=sys_sql_logins_vu_login3 password=123
+SELECT * FROM dbo.sys_sql_logins_vu_prepare_view
 GO
 ~~START~~
-varchar#!#int#!#varbinary#!#char#!#nvarchar#!#int#!#datetime#!#datetime#!#varchar#!#varchar#!#int#!#int#!#bit#!#bit#!#bit#!#varbinary
+varchar#!#nvarchar
+jdbc_user#!#SQL_LOGIN
+sys_sql_logins_vu_login3#!#SQL_LOGIN
 ~~END~~
 
 
-SELECT sys_sql_logins_vu_prepare_func()
+-- tsql
+ALTER ROLE sysadmin add member sys_sql_logins_vu_login1
+GO
+
+-- tsql user=sys_sql_logins_vu_login1 password=123
+SELECT * FROM dbo.sys_sql_logins_vu_prepare_view
+GO
```

Comment:
Please add tests to check other columns of the view like `is_policy_checked`, `is_expiration_checked` etc.